src/components/Authentication/LoginForm.tsx

```diff
@@ -37,7 +37,7 @@ const LoginForm = () => {
       .catch((error) => {
         setState({ state: "error", error: error.data.message });
       });
-  }, [isForgoten, user]);
+  }, [user, login]);
```

Comment:
isn't `isForgoten` overengineering

you either are 
- logged in 
- not logged in 

what is  `isForgoten` state ?